cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -358,8 +355,9 @@ public void dropAllInNamespace(Id.Namespace namespaceId) throws Exception {
       // Note: The trailing "." is crucial, since otherwise nsId could match nsId1, nsIdx etc
       // It's important to keep config table enabled while disabling and dropping  queue tables.
       final String queueTableNamePrefix = String.format("%s.%s.", Id.Namespace.SYSTEM.getId(), queueType);
-      final TableId configTableId = getConfigTableId(namespaceId.getId());
-      tableUtil.deleteAllInNamespace(getHBaseAdmin(), namespaceId, new Predicate<TableId>() {
+      final String hbaseNamespace = tableUtil.getHBaseNamespace(namespaceId);
+      final TableId configTableId = TableId.from(hbaseNamespace, getConfigTableId());
```

Comment:
would it make sense for this call to go through ``HBaseTableUtil`` as well, for standardization?